src/screens/onboarding/components/step-dots/step-dots.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { View } from 'react-native';
+
+import styles from './styles';
+
+type StepDotsProps = {
+  activeDot: number;
+};
+
+const StepDots: React.FC<StepDotsProps> = ({ activeDot }) => {
+  return (
+    <View style={styles.dotsContainer}>
+      <View style={[styles.dot, activeDot === 0 && styles.activeDot]} />
+      <View style={[styles.dot, activeDot === 1 && styles.activeDot]} />
+      <View style={[styles.dot, activeDot === 2 && styles.activeDot]} />
+      <View style={[styles.dot, activeDot === 3 && styles.activeDot]} />
```

Comment:
Move this thing (`activeDot === 3 && styles.activeDot`) to separate `getActiveStyle` function, which can be implemented like this:
```typescript
const getActiveStyle = (dotIndex: number) => {
  if (activeDot === dotIndex) {
    return styles.activeDot;
  }

  return {};
};
```